doc/source/methods.rst

```diff
@@ -0,0 +1,143 @@
+Methods
+=======
+
+Calibration
+-----------
+
+At present, calibration coefficients provided by Andrew Heidinger
+(NOAA) under SCOPE-CM project are applied for all satellites. The current
+version is *PATMOS-x, v2017r1*, including provisional coefficients for MetOp-C.
```

Comment:
Should we show where the actual used version is provided in the code?

Reply:
Yes!